addons/gdUnit3/src/update/GdUnitUpdate.gd

```diff
@@ -125,17 +131,23 @@ func _on_update_pressed():
 		stop_progress()
 		return
 	update_progress("disable GdUnit3 ..")
```

Comment:
I just realized you can probably move the "disable GdUnit3" step all the way to line 163 (right before `# delete the old version at first`), because before that stage the actual add-on directory is not touched. This way you don't even need the `enable_gdUnit()` call in the error case for the extraction 😃